### DISCUSSION TITLE
Buffer is not defined

### DISCUSSION THREAD

**Discussion by halivert** [babel/babel#13852] [Q&A]
I'm using `@babel/standalone` and `@babel/preset-react` in a vite vanilla js project.

In development mode, it works as usual, but in production, the console, throws a strange error:
> Uncaught ReferenceError: Buffer is not defined

```js
ohn=Buffer.isBuffer
```

I don't know why this code throws the error, but persists even if I add `window.Buffer` to my previous code, how can I solve this?

**Comment by s-pofalitov** [comment]
@halivert 
Try it:
https://stackoverflow.com/questions/68707553/uncaught-referenceerror-buffer-is-not-defined
